static/js/settings_config.ts

```diff
@@ -467,12 +467,11 @@ export const expires_in_values = {
         description: $t({defaultMessage: "30 days"}),
         default: false,
     },
-    // Backend support for this configuration is not available yet.
-    // never: {
-    //     value: "never",
-    //     description: $t({defaultMessage: "Never expires"}),
-    //     default: false,
-    // }
+    never: {
+        value: "null",
```

Comment:
I think this should be `null`, not `"null"`.

Reply:
If we keep this value as `null` and not `"null"`, then the value returned by `.val()` for `option` element will be `""`, so I think it would be better to use "null" instead of "".

Reply:
Yeah, OK, that makes sense.

Reply:
Added comments documenting this somewhat unusual looking code :).